.gitignore

```diff
@@ -22,3 +22,4 @@
 .yardoc/
 /doc/
 /failures
+.DS_Store
```

Comment:
Normally for platform specific files, just include that in your `core.excludesFile`, which is described at the lower part of the description here https://git-scm.com/docs/gitignore